src/snowflake/snowpark/file_operation.py

```diff
@@ -282,3 +284,66 @@ def put_stream(
         result_meta = cursor.description
         put_result = result_set_to_rows(result_data, result_meta)[0]
         return PutResult(**put_result.asDict())
+
+    def get_stream(
+        self,
+        stage_location,
+        *,
+        parallel: int = 10,
+        statement_params: Optional[Dict[str, str]] = None,
+        decompress: bool = False,
+    ) -> IO[bytes]:
+        """Downloads the specified files from a path in a stage and expose it through a stream.
+
+        Args:
+            stage_location: The full stage path with prefix and file name, from which you want to download the file.
+            parallel: Specifies the number of threads to use for downloading the files.
+                The granularity unit for downloading is one file.
+                Increasing the number of threads might improve performance when downloading large files.
+                Supported values: Any integer value from 1 (no parallelism) to 99 (use 99 threads for downloading files). Defaults to 10.
+            statement_params: Dictionary of statement level parameters to be set while executing this action. Defaults to None.
+            decompress: Specifies whether to use gzip to decompress file after download. Defaults to False.
+
+        Returns:
+            An `_io.BytesIO` object which points to the downloaded file.
```

Comment:
```suggestion
            An ``BytesIO`` object which points to the downloaded file.
```
I don't think _io is needed